base/inc/AdePT/TrackManager.h

```diff
@@ -0,0 +1,263 @@
+// SPDX-FileCopyrightText: 2022 CERN
+// SPDX-License-Identifier: Apache-2.0
+
+/**
+ * @file TrackManager.h
+ * @brief A track manager using a circular buffer with slot compression capability.
+ * @details The track manager is templated by the track type and holds a circular
+ *          buffer of pre-allocated tracks. It also holds two MParray with track
+ *          slots for the current and next iteration.
+ *
+ *          The compression of the sparse active slots is triggered when the sum of the
+ *          number of used slots and 3 times the number of inflight tracks becomes larger
```

Comment:
should be "2 times the number of inflight tracks" (correctly done in the code below)

Reply:
Fixed